src/components/BuildBaronAndAnnotations/BBComponents.tsx

```diff
@@ -0,0 +1,150 @@
+`;
+
+interface JiraTicketRowProps {
+  jiraKey: string;
+  fields: TicketFields;
+}
+export const JiraTicketRow: React.FC<JiraTicketRowProps> = ({
+  jiraKey,
+  fields,
+}) => {
+  const taskAnalytics = useTaskAnalytics();
+
+  const { data } = useQuery<GetSpruceConfigQuery>(GET_SPRUCE_CONFIG);
+  const spruceConfig = data?.spruceConfig;
+  const jiraHost = spruceConfig?.jira?.host;
+  const url = `https://${jiraHost}/browse/${jiraKey}`;
+  return (
+    <div>
+      <JiraSummaryLink
+        href={url}
+        data-cy={`${jiraKey}`}
+        onClick={() =>
+          taskAnalytics.sendEvent({ name: "Click Jira Summary Link" })
+        }
+      >
+        {jiraKey}: {fields?.summary} {"   "}
+      </JiraSummaryLink>
+
+      <StyledBadge data-cy={`${jiraKey}-badge`} variant="lightgray">
+        {fields?.status.name}
+      </StyledBadge>
+
+      <MetaDataWrapper data-cy={`${jiraKey}-metadata`}>
+        <Disclaimer>
+          Created: {getDateCopy(fields?.created, null, true)}{" "}
+        </Disclaimer>
+        <Disclaimer>
+          Updated: {getDateCopy(fields?.updated, null, true)}{" "}
+        </Disclaimer>
+        <Disclaimer>
+          {fields?.assigneeDisplayName
+            ? `Assignee: ${fields?.assigneeDisplayName}`
+            : "Unassigned"}{" "}
+        </Disclaimer>
+      </MetaDataWrapper>
+    </div>
+  );
+};
+interface AnnotationTicketRowProps {
+  issueKey: string;
+  url: string;
+  source: Source;
+  jiraTicket: JiraTicket;
+}
+
+export const AnnotationTicketRow: React.FC<AnnotationTicketRowProps> = ({
+  issueKey,
+  url,
+  source,
+  jiraTicket,
+}) => {
+  console.log(source);
+  return (
+    <div>
+      <JiraSummaryLink href={url} data-cy={`${issueKey}`}>
+        {issueKey}
+        {jiraTicket?.fields?.summary && `: ${jiraTicket?.fields?.summary}`}
+      </JiraSummaryLink>
+
+      {jiraTicket && (
+        <StyledBadge data-cy={`${issueKey}-badge`} variant="lightgray">
+          {jiraTicket?.fields?.status.name}
+        </StyledBadge>
+      )}
+
+      <MetaDataWrapper data-cy={`${issueKey}-metadata`}>
+        <Disclaimer>
+          {jiraTicket?.fields?.created &&
+            `Created: ${getDateCopy(jiraTicket?.fields.created, null, true)}`}
+        </Disclaimer>
+        <Disclaimer>
+          {jiraTicket?.fields?.created &&
+            `Updated: ${getDateCopy(jiraTicket?.fields.updated, null, true)}`}
+        </Disclaimer>
+        <Disclaimer>
+          {jiraTicket?.fields &&
+            !jiraTicket?.fields?.assigneeDisplayName &&
+            "Unassigned"}{" "}
+          {jiraTicket?.fields?.assigneeDisplayName &&
+            `Assignee: ${jiraTicket?.fields?.assigneeDisplayName}`}
+        </Disclaimer>
+        <Disclaimer>
+          {jiraTicket?.fields?.assignedTeam &&
+            `Assigned Team: ${jiraTicket?.fields?.assignedTeam}`}{" "}
+        </Disclaimer>
+      </MetaDataWrapper>
+    </div>
+  );
+};
+
+const JiraSummaryLink = styled(StyledLink)`
```

Comment:
This will be used many times throughout the page, for related tickets, suspected issues, issues, createdTickets. It's probably worth dedicating this. Is there something specific you had in mind that is very similar?

Reply:
I deleted  
```
 font-size: 14px;
 line-height: 20px;
```
but I need the margin and the weight.

Reply:
@malikchaya2 we can use `BoldStyledLink` to avoid applying the weight style

Reply:
@malikchaya2 It looks like they are not quite the same component